x-pack/plugin/snapshot-repo-test-kit/src/main/java/org/elasticsearch/repositories/blobstore/testkit/BlobAnalyzeAction.java

```diff
@@ -178,7 +178,7 @@ protected void doExecute(Task task, Request request, ActionListener<Response> li
                 throw new IllegalArgumentException("repository [" + request.getRepositoryName() + "] is read-only");
             }
             final BlobStoreRepository blobStoreRepository = (BlobStoreRepository) repository;
-            final BlobContainer blobContainer = blobStoreRepository.blobStore().blobContainer(new BlobPath().add(request.blobPath));
+            final BlobContainer blobContainer = blobStoreRepository.blobStore().blobContainer(blobStoreRepository.basePath().add(request.blobPath));
```

Comment:
We weren't using the provided `base_path` during the repo test kit analysis. I think this could lead to permission problems.